data/campaigns/Under_the_Burning_Suns/utils/abilities.cfg

```diff
@@ -95,12 +95,83 @@ _" This unit is capable of healing those around it, slowing dehydration, and cur
 #enddef
 
 #define ABILITY_FORMATION
-    [dummy]
+	    [defense]
```

Comment:
If we're going to be enabling this, shouldn't the existing implementation be uncommented? (Just search the source files for "defense", you should be able to find it.)